src/include/common/expression_type.h

```diff
@@ -35,6 +35,7 @@ const std::string CAST_TO_BLOB_FUNC_NAME = "BLOB";
 
 // list
 const std::string LIST_CREATION_FUNC_NAME = "LIST_CREATION";
+const std::string LIST_RANGE_FUNC_NAME = "RANGE";
```

Comment:
Not related to this PR but I guess we can start sorting these function names in alphabetical order. Also sort code when you register.